templates/detailItems/adr.html

```diff
@@ -2,10 +2,6 @@
 	<option ng-repeat="option in ctrl.availableOptions" value="{{option.id}}">{{option.name}}</option>
 </select>
 <div>
-	<label for="details-pobox">{{ctrl.t.poBox}}</label>
```

Comment:
Ugh, way to make a spec complicated. :\ P.O. box won’t be used in the majority of cases anyway, similar to extended address.